docs/templates/modules_by_support.rst.j2

```diff
@@ -1,5 +1,8 @@
 .. _@{ slug }@:
 
+{# avoids rST "isn't included in any toctree" errors for module index docs #}
+:orphan:
+
```

Comment:
Does modules_by_category.rst.j2 now have the :orphan: role as well?  I think these are similar so probably if modules_by_category ends up in toctrees, this should as well.

Reply:
The `modules_by_category` pages don't trigger a warning, so they must be in a toctree somewhere. Or sphinx is just messing with me. Either way, it ain't broke, so I'll leave it as it is.

Reply:
My mistake. The `modules_by_category` pages do, in fact, trigger warnings. Fixing now

Reply:
Fixed.

Reply:
<nod>  that suggests to me that these should be in a toctree as well.  ie:  the warning is about a valid issue.